app/javascript/packs/application/redux/points/actions.test.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { createStore, combineReducers, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
+import { listPoints, createPoint } from "./actions";
+import api from "../../api";
+
+describe("listPoints()", () => {
+  it("calls the correct dispatch and returns data when successful", async () => {
+    const dispatch = jest.fn();
+    const point = { id: 1, title: "Test Point" };
+
+    jest.spyOn(api, "listPoints").mockResolvedValue({
+      ok: true,
+      data: [point]
+    });
+
+    await listPoints()(dispatch);
```

Comment:
Do you mean to test `listPoints` with `tourId === undefined` here?  Maybe I'm missing something about how `api.listPoints` works.